R/shapley.R

```diff
@@ -45,29 +45,24 @@ weight_matrix <- function(X, use_shapley_weights_in_W = TRUE, normalize_W_weight
 }
 
 
-#' Create an explainer object
+#' Create an explainer object with Shapley weights for test data.
 #'
-#' @details TODO: Write a better description of this function.
+#' @inheritParams global_arguments
 #'
-#' @param x A numeric matrix or data.frame. Contains the variables used for training the model
+#' @param x An \code{ntrain x p} numeric matrix or data.frame, where \code{p = ncol(x)} (total number of explanatory variables).Contains the variables used for training the model
 #' (i.e. the explanatory variables). Note that the response variable should not be part of
 #' \code{x}.
 #'
-#' @param n_combinations Integer. The number of feature combinations to sample. If ´\code{NULL},
-#' the exact method is used and all combinations are considered. The maximum number of
-#' combinations equlas \code{2^p}, where \code{p = ncol(x)} (total number of explanatory variables).
+#' @param model The model whose predictions we want to explain.
 #'
-#' @details
-#' TODO: Write about missing values and categorical variables
```

Comment:
Should this be done? Or is it mentioned elsewhere that we only deal with continuous variables ?

Reply:
It is mentioned in `README.md`, but we should add some additional checks on the data. I.e. if the user pass training data that is not numerical or contains missing values, a meaningful error message should be returned.